### PR TITLE
Add Linux Support (WIP)

### PR DESCRIPTION
**Pull request by TheLarkInn**
This PR adds additional columns for Linux and their specific feature parity in the platform support page. This PR is still work in progress as I need to have specific PM's approve the individual features that are suppported.

### CODE REVIEW

**Comment by TheLarkInn** [comment]
This is ready for code review. I have received +1's from stakeholders on the changes made.

**Comment by TheLarkInn** [comment]
cc @Reezaali

File: edgeenterprise/microsoft-edge-platform-feature-matrix.md [hunk 1]
```diff
@@ -26,42 +26,42 @@ The following tables summarize feature support for the Windows and macOS platfor
...
+|Enterprise New Tab Page|Yes|Yes|Yes|Yes|Yes|[New Tab Page](https://blogs.windows.com/msedgedev/2020/10/29/enterprise-new-tab-page-my-feed/) |
+|IE mode|Yes|Yes|Yes|No|No|[IE mode](/deployedge/edge-ie-mode#prerequisites)|
+|Kiosk Mode|Yes|No|No|No|No|[Kiosk Mode](/deployedge/microsoft-edge-configure-kiosk-mode)|
+|Microsoft Search in Bing|Yes|Yes|Yes|Yes|Yes|[Intelligent Search in Bing](https://www.microsoft.com/edge/business/intelligent-search-with-bing) |
+|PDF Reader|Yes|Yes|Yes|Yes|Yes|[PDF Reader](/deployedge/microsoft-edge-pdf) |
+|Shopping|Yes|Yes|Yes|Yes|Yes|[Shopping](https://techcommunity.microsoft.com/t5/articles/introducing-shopping-with-microsoft-edge/m-p/1870080) |
+|Sleeping Tabs|Yes|Yes|Yes|Yes|Yes|[Feature overview](/deployedge/microsoft-edge-relnote-stable-channel)<br>[Latest Blog Post](https://blogs.windows.com/msedgedev/2021/03/04/edge-89-performance/)<br>[Group Policies](/deployedge/microsoft-edge-policies#sleeping-tabs-settings)|
+|Sync|Yes|Yes|Yes|Yes|Yes|[Enterprise Sync](/deployedge/microsoft-edge-enterprise-sync) |
```
Comment (TheLarkInn) [inline edgeenterprise/microsoft-edge-platform-feature-matrix.md:62]: Fix this back to No. We can update this when sync lands/

File: edgeenterprise/microsoft-edge-platform-feature-matrix.md [hunk 2]
```diff
@@ -26,42 +26,42 @@ The following tables summarize feature support for the Windows and macOS platfor
...
-|Sync|Yes|Yes|Yes|Yes| [Enterprise Sync](/deployedge/microsoft-edge-enterprise-sync) |
-|Version Rollback|Yes|Yes|Yes|No|[Version rollback](/deployedge/edge-learnmore-rollback) |
-|Vertical Tabs|Yes|Yes|Yes|Yes| |
+|Additional features|Win 10|Win 8.1|Win 7|macOS|Linux|URL|
+|-------------------|------|-------|-----|-----|-----|---|
+|Collections|Yes|Yes|Yes|Yes|Yes|[Collections](https://blogs.windows.com/msedgedev/2019/12/09/improvements-collections-sync-microsoft-edge/) |
+|Enterprise New Tab Page|Yes|Yes|Yes|Yes|Yes|[New Tab Page](https://blogs.windows.com/msedgedev/2020/10/29/enterprise-new-tab-page-my-feed/) |
+|IE mode|Yes|Yes|Yes|No|No|[IE mode](/deployedge/edge-ie-mode#prerequisites)|
+|Kiosk Mode|Yes|No|No|No|No|[Kiosk Mode](/deployedge/microsoft-edge-configure-kiosk-mode)|
+|Microsoft Search in Bing|Yes|Yes|Yes|Yes|Yes|[Intelligent Search in Bing](https://www.microsoft.com/edge/business/intelligent-search-with-bing) |
```
Comment (TheLarkInn) [inline edgeenterprise/microsoft-edge-platform-feature-matrix.md:58]: Switch back until AAD is landed

Comment (dan-wesley) [reply edgeenterprise/microsoft-edge-platform-feature-matrix.md:58]: @TheLarkInn Not sure what you want to switch back until AAD is landed. AFAIK, we should be good to go.

File: edgeenterprise/microsoft-edge-platform-feature-matrix.md (hunk 1 above)
Comment (dan-wesley) [reply edgeenterprise/microsoft-edge-platform-feature-matrix.md:62]: @TheLarkInn Sync should be ok.

**Review by dan-wesley — APPROVED** [review]

**Pull request merged**